azurecontainerapps.ts

```diff
@@ -573,15 +563,15 @@ export class azurecontainerapps {
             return;
         }
 
-        if (this.shouldUseUpdateCommand && !createOrUpdateContainerAppWithUp) {
+        if (this.shouldUseUpdateCommand && !this.shouldCreateOrUpdateContainerAppWithUp) {
```

Comment:
nit: we should see if the conditionals for each of these variables are specific enough to distinguish them a part; _i.e._, it would be great if this given conditional was just

```typescript
if (this.shouldUseUpdateCommand)
```

since it would imply that some conditional set for `this.shouldCreateOrUpdateContainerAppWithUp` was not met and it's `false`, so we wouldn't need the additional `&& !this.shouldCreateOrUpdateContainerAppWithUp` piece.

Reply:
Thanks @cormacpayne! Is this something we can look into post-Ignite?